csv-validator-ui/src/main/scala/uk/gov/nationalarchives/csv/validator/ui/CsvValidatorUi.scala

```diff
@@ -145,12 +145,23 @@ object CsvValidatorUi extends SimpleSwingApplication {
     var badLines = 0
     var truncated = false
 
-    def rowCallback(row: ValidatedNel[FailMessage, Any]): Unit = row match {
+    val maxNumOfLines = Try(potentialMaxNumOfLines.toInt) match {
+      case Success(number) if number > 0 => number
+      case _ =>
```

Comment:
I think if we error finding the limit, we should, perhaps, just go ahead with the current default limit of 2000 instead of setting it to zero. Maybe get Steve's opinion as well.

Reply:
The reason I set it to 0 here is so that on line [154](https://github.com/digital-preservation/csv-validator/pull/527/files/80d547e9d4a121a18d248d4a056fbd6ac97083f1#diff-3e28ce602b0cfebcb44c219baf94c2402228bbafd9567ea222a07f11b388aaa4R152), it can do the conditional check `maxNumOfLines > 0` and not run the validation (after it sends the error message on line 151). Wouldn't it be better to just return an error rather than running the validation?

Reply:
My thinking is, it becomes more restrictive than previous version, also if that setting is  missing, the application can still carry on and succeed, hence the comment. Perhaps get Steve's opinion as well.